masterlist.yaml

```diff
@@ -29872,3 +29872,7 @@ plugins:
 # Load Ordinator after other perk mods so it has precedence.
   - name: 'Ordinator - Perks of Skyrim.esp'
     priority: 1
+  - name: 'Darkend.esp'
```

Comment:
Is this not a typo? The mod is big so I don't want to download it just to check myself.